teams/teams-ps/teams/Set-CsTeamsComplianceRecordingPolicy.md

```diff
@@ -0,0 +1,361 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+online version: https://learn.microsoft.com/powershell/module/skype/set-csteamscompliancerecordingpolicy
+applicable: Skype for Business Online
+title: Set-CsTeamsComplianceRecordingPolicy
+schema: 2.0.0
+manager: nakumar
+author: aditdalvi
+ms.author: aditd
+ms.reviewer:
+---
+
+# Set-CsTeamsComplianceRecordingPolicy
```

Comment:
the article for this cmdlet already exists https://learn.microsoft.com/en-us/powershell/module/skype/set-csteamscompliancerecordingpolicy?view=skype-ps

Reply:
This article is for the modern implementation that lives under the "teams" path. We can either remove the article that lives under the "skype" path or keep both.